ui/e2e/src/tests/markdownPanel.spec.ts

```diff
@@ -34,7 +34,7 @@ test.describe('Dashboard: Markdown Panel', () => {
       const markdownPanel = dashboardPage.getPanel(panelName);
       await markdownPanel.isLoaded();
 
-      await happoPlaywright.screenshot(page, markdownPanel.container, {
+      await happoPlaywright.screenshot(page, markdownPanel.parent, {
```

Comment:
The width/height live on the container, so previously the screenshots were filling the whole page. This was way more noticeable when working on the canvas visualizations because happo turns them into images, so they weren't stretching. Moving to the parent, keeps them the size we intend to test.

Reply:
I swapped them to a 3 across size, so they're a little bigger now and probably at a common size people might use them in a dashboard. Does that look better to you?

Reply:
Oh where should I go to look, it looks like the Happo link above still points to old ones

Reply:
Oops, sorry about there. Here's the new one. https://happo.io/a/1009/p/1488/compare/9e577592bcdc75699bd55ad53ab0026e5e80ecf6/58c5e98ef29df87f73f159015136533ce65ea65c

You can also find it by clicking the "details" link on the happo check 
<img width="848" alt="image" src="https://user-images.githubusercontent.com/396962/214976664-03d856b1-9be8-47bd-a1e6-eb443b71ec46.png">

Reply:
Looks great, I like it!